### PR TITLE
Add Hovrly.app v2.4.5

### PR DESCRIPTION
**Pull request by tarutin**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

**Comment by tarutin** [comment]
This is `brew audit --cask --online <cask>` only for **existing casks**? Right?

```› brew audit --cask --online hovrly
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
==> Downloading https://github.com/tarutin/hovrly/releases/download/v2.4.5/Hovrly-2.4.5-universal.dmg
Already downloaded: /Users/tarutin/Library/Caches/Homebrew/downloads/238d600ab778e59175e51ca26a1aefa00c3de2ca71aecf8ae447f58b9d989d25--Hovrly-2.4.5-universal.dmg
audit for hovrly: failed
 - Version '2.4.5' differs from '' retrieved by livecheck.
 - Version '2.4.5' differs from '' retrieved by livecheck.
hovrly
  * Version '2.4.5' differs from '' retrieved by livecheck.
Error: 1 problem in 1 cask detected.```

File: Casks/hovrly.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+cask "hovrly" do
+  version "2.4.5"
+  sha256 "2cd4d3fc696a7a59446fea3c0a1d9f77b23cfed8bcd53c6992ffb5dc78711b88"
+
+  url "https://github.com/tarutin/hovrly/releases/download/v#{version}/Hovrly-#{version}-universal.dmg", verified: "github.com/tarutin/hovrly/"
+  name "Hovrly"
+  desc "Best partner for disctributed teams"
```
Comment (krehel) [inline Casks/hovrly.rb:7]: Please update this - desc should provide a brief purpose of the application

**Review by krehel — CHANGES_REQUESTED** [review]

File: Casks/hovrly.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  homepage "https://hovrly.com/"
+
+  app "Hovrly.app"
+
+  # No zap stanza required
```
Comment (krehel) [inline Casks/hovrly.rb:12]: A `zap` stanza is required here. 
```
~/Library/Application Support/Hovrly
~/Library/Preferences/com.treasy.hovrly.plist
~/Library/Saved Application State/com.treasy.hovrly.savedState
```

File: Casks/hovrly.rb [hunk 3]
```diff
@@ -0,0 +1,13 @@
+cask "hovrly" do
+  version "2.4.5"
+  sha256 "2cd4d3fc696a7a59446fea3c0a1d9f77b23cfed8bcd53c6992ffb5dc78711b88"
+
+  url "https://github.com/tarutin/hovrly/releases/download/v#{version}/Hovrly-#{version}-universal.dmg", verified: "github.com/tarutin/hovrly/"
```
Comment (krehel) [inline Casks/hovrly.rb:5]: `url` and `verified` should be on separate lines

**Review by krehel — CHANGES_REQUESTED** [review]

File: Casks/hovrly.rb (hunk 1 above)
Comment (tarutin) [reply Casks/hovrly.rb]: done

File: Casks/hovrly.rb (hunk 2 above)
Comment (tarutin) [reply Casks/hovrly.rb]: done

File: Casks/hovrly.rb (hunk 3 above)
Comment (tarutin) [reply Casks/hovrly.rb]: done

Comment (tarutin) [reply Casks/hovrly.rb]: `Error: Layout/ArgumentAlignment: Align the arguments of a method call if they span more than one line.`

Are U sure?)

**Review by tarutin — COMMENTED** [review]
done

**Comment by krehel** [comment]
Cheers @tarutin, looks good!  Thanks for your submission!

**Review by krehel — APPROVED** [review]

**Pull request merged**